src/writeData/subscriptions/components/JsonParsingForm.tsx

```diff
@@ -234,10 +261,15 @@ const JsonParsingForm: FC<Props> = ({formContent, updateForm}) => {
                   id={r}
                   value={r}
                   onClick={() => {
+                    event(
+                      'added json parsing rule',
+                      {ruleType: r},
+                      {feature: 'subscriptions'}
+                    )
                     setRule(r)
                   }}
                   selected={rule === r}
-                  testID={`json-parsing-add-rule-${1}`}
+                  testID={`json-parsing-add-rule-${key}`}
```

Comment:
saw this and figured it should be `key` but let me know if the `1` was intentional

Reply:
not intentional :)